themes/aframe/source/css/home.styl

```diff
@@ -17,9 +17,6 @@ ul
 .home
   display flex
   flex-wrap wrap
-  padding 30px
```

Comment:
I was going to make the layout borderless with padding within the content sections itself.   looks jank now, but will fix.